src/es2015/text.js

```diff
@@ -0,0 +1,4 @@
+module.exports = () => {
+	// https://mathiasbynens.be/notes/es-unicode-property-escapes#emoji
+	return /<% emojiSequenceUnicode %>|\p{Emoji_Modifier_Base}\p{Emoji_Modifier}?|\p{Emoji_Presentation}|\p{Emoji}\uFE0F?/gu;
+};
```

Comment:
Hmm, can we not duplicate these two files? The input is the same, it’s just that we output in two different formats.